M2/Macaulay2/packages/Style.m2

```diff
@@ -3,16 +3,184 @@ newPackage( "Style",
      AuxiliaryFiles => true,
      Headline => "style sheets and images for the documentation",
      Keywords => {"Documentation"},
-     Version => "1.0"
+     Version => "1.1"
      )
 
+---------------------------------------------
+-- grammar generation code adapted from    --
+-- M2/Macaulay2/editors/make-M2-symbols.m2 --
+---------------------------------------------
+
+export {"generateGrammar"}
+
+importFrom(Core, "sortBy")
+
+-- TODO: Move these two elsewhere:
+Function and Function := (f, g) -> s -> f s and g s
+Function or  Function := (f, g) -> s -> f s or  g s
```

Comment:
Do you want to actually move these to Core while you're moving them? Maybe also add `not Function` and `Function xor Function`. Especially the former would be super useful for things like this:
```m2
i1 : not Function := f -> x -> not f x

i2 : select({0,1,0,2,0,3}, not zero)

o2 = {1, 2, 3}
```

Reply:
Might as well!